### PR TITLE
Avoid ^ for exponentiation

### PR DESCRIPTION
**Pull request by iugo**
#### Summary

Use ** instead of ^.

#### Motivation

To avoid confusion with XOR.

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

**Comment by teoli2003** [comment]
So for a long time, JS had only `pow()` and no exponentiation operator. Using `^` wasn't really problematic.

I'm torn on this one. 

What do you think, @Josh-Cena ? Should we ask this in a discussion to have more opinions? This will definitively have an impact on numerous other places on MDN.

**Comment by Josh-Cena** [comment]
What about using Math ML, or use a simple `<sup>`? I think `**` is less readable and it's not meant to be code here anyway, so I think we should definitely remove the backticks.

**Comment by teoli2003** [comment]
I think we should:
1. if it is not code but an explanation, do not have back-ticks and use `<sup>`;
2. if it is code, use `**`.

So avoid `^` and `**` outside code.

I think we are in case 1 here.

File: files/en-us/web/javascript/reference/global_objects/number/max_safe_integer/index.md [hunk 1]
```diff
@@ -11,15 +11,16 @@ browser-compat: javascript.builtins.Number.MAX_SAFE_INTEGER
 ---
 {{JSRef}}
 
-The **`Number.MAX_SAFE_INTEGER`** constant represents the maximum safe integer in JavaScript (`2^53 - 1`).
+The 
+`Number.MAX_SAFE_INTEGER`** constant represents the maximum safe integer in JavaScript (`2 ** 53 - 1`).
```
Comment (teoli2003) [inline files/en-us/web/javascript/reference/global_objects/number/max_safe_integer/index.md:15]:
```suggestion
The `Number.MAX_SAFE_INTEGER`** constant represents the maximum safe integer in JavaScript (2<sup>53</sup – 1).
```

**Comment by iugo** [comment]
I'm asking this question because the `^` symbol has two completely different meanings in math and programming.

https://stackoverflow.com/questions/4843304/why-is-my-power-operator-not-working

Hopefully there is a better way to avoid this ambiguity.

File: files/en-us/web/javascript/reference/global_objects/number/max_safe_integer/index.md [hunk 2]
```diff
@@ -11,15 +11,16 @@ browser-compat: javascript.builtins.Number.MAX_SAFE_INTEGER
...
 
 For larger integers, consider using {{jsxref("BigInt")}}.
 
 {{EmbedInteractiveExample("pages/js/number-maxsafeinteger.html")}}{{js_property_attributes(0, 0, 0)}}
 
 ## Description
 
-The `MAX_SAFE_INTEGER` constant has a value of `9007199254740991` (9,007,199,254,740,991 or \~9 quadrillion). The reasoning behind that number is that JavaScript uses [double-precision floating-point format numbers](https://en.wikipedia.org/wiki/Double_precision_floating-point_format) as specified in [IEEE 754](https://en.wikipedia.org/wiki/IEEE_floating_point) and can only safely represent integers between `-(2^53 - 1)` and `2^53 - 1`.
+The `MAX_SAFE_INTEGER` constant has a value of `9007199254740991` (9,007,199,254,740,991 or \~9 quadrillion). The reasoning behind that number is that JavaScript uses [double-precision floating-point format numbers](https://en.wikipedia.org/wiki/Double_precision_floating-point_format) as specified in [IEEE 754](https://en.wikipedia.org/wiki/IEEE_floating_point) and can only safely represent integers between `-(2 ** 53 - 1)` and `2 ** 53 - 1`.
```
Comment (teoli2003) [inline files/en-us/web/javascript/reference/global_objects/number/max_safe_integer/index.md:23]:
```suggestion
The `MAX_SAFE_INTEGER` constant has a value of `9007199254740991` (9,007,199,254,740,991 or \~9 quadrillion). The reasoning behind that number is that JavaScript uses [double-precision floating-point format numbers](https://en.wikipedia.org/wiki/Double_precision_floating-point_format) as specified in [IEEE 754](https://en.wikipedia.org/wiki/IEEE_floating_point) and can only safely represent integers between -(2<sup>53</sup – 1) and 2<sup>53</sup – 1.
```

**Comment by Josh-Cena** [comment]
I think it should be a one-time effort to fix it everywhere. We have about 20 pages with this issue. I'm going to commit to this branch.

**Comment by teoli2003** [comment]
(I think MathML is a bit overkill for this, at least for the time being).

**Comment by Josh-Cena** [comment]
I realized we have this guide, though: https://developer.mozilla.org/en-US/docs/MDN/Contribute/Markdown_in_MDN#superscript_and_subscript We should reconsider that suggestion.

**Review by teoli2003 — APPROVED** [review]
Let's go like this 😃

**Comment by iugo** [comment]
I noticed some previous related discussions. https://github.com/mdn/content/issues/4578#issuecomment-830607335

---

Maybe should remove particular for exponentiation.

**Comment by teoli2003** [comment]
Yes, that discussion happened when we moved to Markdown last year. Before we stored our pages in HTML.

We tried to minimize the use of HTML tags.

We decided to keep the `<sup>` tag when useful.

**Pull request merged**